### PR TITLE
Fixed configuration for all documentation

### PR DESCRIPTION
**Pull request by rogergros**

### CODE REVIEW

File: src/Elcodi/Bundle/AttributeBundle/README.md [hunk 1]
```diff
@@ -76,6 +84,32 @@ class AppKernel extends Kernel
 }
 ```
 
+The default configuration values for the bundle are are:
```
Comment (javiereguiluz) [inline src/Elcodi/Bundle/AttributeBundle/README.md]: typo here: `are are:`

File: src/Elcodi/Bundle/GeoBundle/README.md [hunk 2]
```diff
@@ -44,7 +44,14 @@ bundle, you'll find info on this [symfony documentation page][4]
...
 from
 [elcodi/geo-bundle packagist](https://packagist.org/packages/elcodi/geo-bundle)
-by just adding a line in your composer.json
+by just executing the following line
+
+``` bash
+$ composer require "elcodi/geo-bundle:~0.5.*"
+```
+
+You can also do it manually by adding a line in your ``composer.json`` file
```
Comment (javiereguiluz) [inline src/Elcodi/Bundle/GeoBundle/README.md]: unnecessary double backticks here: they are used in reStructuredText doc, but this is Markdown

File: src/Elcodi/Bundle/GeoBundle/README.md [hunk 3]
```diff
@@ -83,6 +90,47 @@ class AppKernel extends Kernel
...
+
+``` yaml
+your_route_name:
+    resource: "@ElcodiGeoBundle/Resources/config/routing.yml"
+    prefix: /geo
+```
+
+> You'll need to do this to use GEO as an API
+
+The default configuration values for the bundle are are:
```
Comment (javiereguiluz) [inline src/Elcodi/Bundle/GeoBundle/README.md]: duplicate `are are` here too

**Comment by rogergros** [comment]
Thanks @javiereguiluz I'm removing all the "are are" and the unnecessary double backticks in all files :+1:

**Pull request merged**